project-base/app/getEnvironment.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Shopsys;
+
+require_once __DIR__ . '/autoload.php';
+
+file_put_contents("php://output", Environment::getEnvironment(false));
```

Comment:
why the `false` value for `$console` parameter? You are calling this from the console only so I am confused now